releases/patch-releases.md

```diff
@@ -82,6 +82,7 @@ End of Life for **1.19** is **2021-09-30**
 | PATCH RELEASE | CHERRY PICK DEADLINE | TARGET DATE |
 |--- |--- |--- |
 | 1.19.10       | 2021-04-09           | 2021-04-14  |
+| 1.19.10       | 2021-04-09           | 2021-04-14  |
```

Comment:
@cpanato i think you have a duplicate here

Reply:
opss :) i'm blind

thanks for the catch

Reply:
fixed